src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
> Yes and no. The sentence here is true with and without the subclass.

Try tasking the concept of an AliasGroup on it's own. Forget about all that is build on top of it. Where in this concept is a "request" present? This implies a request is being made, for which presumably the AliasGroup is the result. And while there definitely usecases in which that can happen, there are also those in which there is no such request. Hence "request" is not part of the AliasGorup concept. Please drop that part from the doc if you put it in this class.

Concrete example: suppose I deserialize an entity from the Wikidata API into a PHP Entity. That has a list of AliasGroup. Yet there is no notion of a language being requested there.

> actual language of the aliases

What does this "actual" mean exactly? "original" language? I'm not sure the above issue is absent here, since it seems to again depend on context.

Concrete example: what I have a place where I do not want to differentiate between original and fallback language, and want to construct an AliasGroup with a fallback language as language?